Makefile

```diff
@@ -28,7 +28,7 @@ help:
 
 .PHONY: clean
 clean:
-	rm -rf $(BUILDDIR)/*
+	rm -rf $(BUILDDIR)/* ./source/plugins/available
```

Comment:
Is this necessary? The mock extension hook should clean everything up, right?

Reply:
I think it would be sensible for `make clean` to clean up all possible known side effects.  If the build fails partway through during local development for some reason, it would be handy to have this.